### PR TITLE
[feature] add external table grant in

### PR DESCRIPTION
**Pull request by fevin86**
This introduce external table grant.

### CODE REVIEW

**Review by ryanking — CHANGES_REQUESTED** [review]
Also

* `go fmt` needs to be run.
* `main` has been upgraded to use the provider sdk v2, this will need to be updated as well

File: pkg/resources/external_table_grant.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+func CreateExternalTableGrant(data *schema.ResourceData, meta interface{}) error {
+	var (
+		externalTableName string
+		schemaName        string
+	)
+	if _, ok := data.GetOk("external_table_name"); ok {
+		externalTableName = data.Get("external_table_name").(string)
+	} else {
+		externalTableName = ""
+	}
```
[the remark this answers is not in the document]
Comment (fevin86) [reply pkg/resources/external_table_grant.go:97]: Thanks for picking it up. I actually just did a copy/paste from the table grant with necessary change. I will clean this up.

File: pkg/resources/external_table_grant.go [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+	if _, ok := data.GetOk("schema_name"); ok {
+		schemaName = data.Get("schema_name").(string)
+	} else {
+		schemaName = ""
+	}
```
[the remark this answers is not in the document]
Comment (fevin86) [reply pkg/resources/external_table_grant.go:102]: same as above comment :)

**Comment by etsheks** [comment]
@ryanking can you please have another look at this PR?

**Comment by fevin86** [comment]
@ryanking can you please have another look at this PR?

**Comment by alldoami** [comment]
@fevin86 could you resolve the conflicts?

**Comment by fevin86** [comment]
This can be closed due to another pull request merged.

**Pull request closed without merging**